keyboards/maxr1998/pulse4k/keymaps/default/keymap.c

```diff
@@ -0,0 +1,33 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+enum layers {
+  DEFAULT
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [DEFAULT] = {
+        { KC_END,  KC_UP,   KC__MUTE },
+        { KC_LEFT, KC_DOWN, KC_RGHT  }
+    }
+};
+
+void matrix_init_user(void) {
+}
+
+void encoder_one_update(bool clockwise) {
+    if (clockwise) {
+        tap_code(KC_PGDN);
+    } else {
+        tap_code(KC_PGUP);
+    }
+}
+
+void encoder_two_update(bool clockwise) {
+    if (clockwise) {
+        tap_code(KC__VOLUP);
```

Comment:
These two keycodes actually only work on macOS - you might want to use `KC_VOLU` and `KC_VOLD` here instead, as they are more universal.

Reply:
I know, however, the "universal" ones didn't work on my Linux system (with i3), interestingly, they weren't even registered in `xev`.. Is there any flag I'd have to set for them to work?

Reply:
Ah, yeah. You need `EXTRAKEY_ENABLE = yes` in your rules.mk.

Reply:
Yes, actually.

You need to set `EXTRAKEY_ENABLE = yes` in the keyboard or keymap's rules.mk file.

Reply:
Oh, I see! Thanks a lot, will do then!